MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ReflectiveSerializer.cs

```diff
@@ -127,6 +133,11 @@ protected internal override void Initialize(IntermediateSerializer serializer)
             }
         }
 
+        public override bool CanDeserializeIntoExistingObject
+        {
+            get { return TargetType.IsClass && TargetType.BaseType != null; }
```

Comment:
This is a bit of a guess - it makes all our tests pass, but we might just be missing test cases.